src/adapter/activateDaffodilDebug.ts

```diff
@@ -209,19 +259,43 @@ export function activateDaffodilDebug(
         createDebugRunFileConfigs(resource, 'run', 'execute')
       }
     ),
-    vscode.commands.registerCommand('extension.dfdl-debug.copyTDML', (_) => {
-      // Ask for destination path
-      // Copy file in /tmp to destination path
-      // TDMLConfig.path should not be used here because that only matters when sending to the server
-      // We could make it so that if someone wants to specify the path and set the action to 'copy', but
-      //   that doesn't make a whole lot of sense.
-      let targetResource = vscode.commands.executeCommand(
-        'extension.dfdl-debug.getTDMLPath'
```

Comment:
Ok, I think I see one of the bugs: there was no `getTDMLPath` command registered. But I still don't know why this needs to be a user command, it could just be a function.

I think you followed the--to my eyes weird--pattern, where a command was declared but not defined. It should just be a function and not declared as a command.